### ISSUE TITLE
Is it possible to hide stacktraces?

### ISSUE THREAD

**Issue by DarrenTsung** [getsentry/sentry-rust#96]
Hi, thanks for the great product! 

In order to integrate the Sentry Rust SDK with our existing Rust app, I created a custom `Append`er (for the fork of `log4rs` we're on) that filters Warnings / Errors to Sentry. Unfortunately this adds 5-6 lines of identical stacktraces for every message that is reported, which I believe is the cause of bad grouping (unrelated events) for our issues. 

The stacktraces look like:
```
...
0x563390102148
...::logging::sentry_appender::event_from_record sentry_appender.rs:38
<unknown>
0x563390102148
<...::logging::sentry_appender::SentryAppender as log4rs::append::Append>::append sentry_appender.rs:19
log4rs
0x563390332cc6
log4rs::Appender::append lib.rs:249
log4rs
0x563390332cc6
log4rs::ConfiguredLogger::log lib.rs:226
<unknown>
0x563390332cc6
<log4rs::Logger as log::Log>::log lib.rs:345
log
0x563390729fbb
log::__log lib.rs:968
```

I noticed in the documentation that it was possible to hide irrelevant stacktraces to improve the grouping, see: https://docs.sentry.io/data-management/rollups/?platform=rust.

```
If you modify your stacktrace by introducing a new level through the use of decorators, your stacktrace will change and so will the grouping. For this matter many SDKs support hiding irrelevant stack trace frames. For instance the Python SDK will skip all stack frames with a local variable called __traceback_hide__ set to True).
```

Is this possible with the Rust SDK? Additionally, if you have any alternative suggestions, that would be helpful.

**Comment by mitsuhiko** [comment]
Yes, there are ways. In the Rust SDK you can modify the `extra_border_frames` option on the client options. If you add the name of a function that is the beginning of the junk in the stacktraces that is chopped off. So for instance if you add `log::__log` everything above will be removed.  Hope that helps.

**Comment by mitsuhiko** [comment]
Also additionally if you don't want the stacktrace at all you can just pass `false` to `event_from_record`.
